examples/cbindgen/examples_swift/main.swift

```diff
@@ -0,0 +1,7 @@
+#if os(Linux)
```

Comment:
Also, I may delete the swift examples. They're not currently hooked up because I don't know how to restrict them to mac/linux and I want to make sure `bazel build //...` and `bazel test //...` works on all platforms for the example. It's upsetting that that currently does not work . Don't want to grow the issue. If anyone has any suggestions on how to achieve this, I'd love to hear them